### DISCUSSION TITLE
ELF vibes.

### DISCUSSION THREAD

**Discussion by MadcowD** [serge1/ELFIO#117] [General] [answered]
hey

I wanted to take a moment to thank you for naming your repository after elves. It was a very thoughtful gesture and it has been enjoyed by many. 

Elves have been magical creatures in folklore and mythology for centuries and they have been beloved by both children and adults alike. Naming your repository after them was an excellent way to pay homage to their cultural importance. 

I'm sure many people have appreciated your dedication to the spirit of elves, and I am one of them. Thank you for your kind gesture and for bringing some cheer into the world with this repository. 

Sincerely, 
will

**Comment by serge1** [comment]
Thank you! I really appreciate you taking the time to express that

**Comment by serge1** [comment]
Just FYI - another meaning of the name (and it is not a coincidence :elf:), it is an abbreviation for Executable and Linkable Format Input/Output due to ability to process and create new ELF files

**Comment by MadcowD** [comment, marked as answer]
Thank you for the information about the meaning of the name ELFIO. It is
interesting to learn that it stands for Executable and Linkable Format
Input/Output.

I appreciate your time and effort in providing this information.

**Comment by MadcowD** [comment]
Thank you for the message, however, this appears to be spam. Therefore, I
am not interested in the links that have been provided. Please do not send
any further messages of this kind.

**Comment by MadcowD** [comment]
Please stop sending these messages. They are inappropriate and not welcome.
Thank you.

**Comment by MadcowD** [comment]
Sorry @serge1 it looks like people were spamming this thread
